dags/hivemind_etl_helpers/tests/integration/test_gdrive_loader.py

```diff
@@ -184,11 +184,10 @@ def test__load_by_folder_id(self, mock_load_data):
                 text="Option 1: Keep it super casual",
             ),
         ]
-        self.mock_loader = Mock()
-        mock_load_data.return_value = mock_data
+        mock_get.return_value = mock_data
         loader = GoogleDriveLoader(client_config=self.mock_client_config)
-        result = loader._load_from_folders(folder_id=folder_id)
-        self.assertEqual(len(result), 2)
+        result = loader._load_from_folders(folder_ids=folder_ids)
+        assert result[0].id_ == mock_data[0].id_
```

Comment:
Why we're removing the assertion of length?
It is very good to do assertions on the values of the result and I insist on keeping it but could you tell me why we're removing the `self.assertEqual(len(result), 2)` from this line?

Note: Please change the `assert [value] == [value2]` to `self.assertEqual([value], [value2])` as we're inheriting from TestCase.unittest and it fully support many kind of assertions.

Reply:
I didn't know it was important to use the length. will fix and push.